cms/envs/common.py

```diff
@@ -2098,6 +2098,12 @@
 # .. toggle_creation_date: 2016-06-30
 ENABLE_COMPREHENSIVE_THEMING = False
 
+# .. setting_name: CUSTOM_RESOURCE_TEMPLATES_DIRECTORY
+# .. setting_default: None
+# .. setting_description: Path to existing directory containing resource templates.
```

Comment:
Just being nitpicky here: Can you add a bit more detail to the description? Looking from an outsiders perspective this is a somewhat cryptic variable that allows customizing something. What are the `resource templates` for? Who is using them? Are there any limitations?

Reply:
Agreed. I'll add more detail to the descriptions.